routers/web/user/setting/account.go

```diff
@@ -108,7 +108,7 @@ func EmailPost(ctx *context.Context) {
 			return
 		}
 		if ctx.Query("id") == "PRIMARY" {
-			if ctx.User.IsActive {
```

Comment:
Now all emails are stored in email_address table, we should get primary email address from that table if id is primary

Reply:
Do you mean remove the Email and IsActive fields from the User table? I agree but I think that could/should be another PR - either in 1.16 or straight after this one is merged.